package.json

```diff
@@ -33,6 +33,7 @@
     "autoprefixer": "^6.5.3",
     "babel-cli": "^6.14.0",
     "babel-core": "^6.13.2",
+    "babel-plugin-transform-object-assign": "^6.22.0",
```

Comment:
J'obtiens l'erreur suivante : 

> ERROR in ./~/react-cropper/dist/react-cropper.js
Module build failed: ReferenceError: Unknown plugin "transform-object-assign" specified in "/Users/bastien/Sites/kitten/spec/dummy/client/node_modules/react-cropper/package.json" at 0, attempted to resolve relative to "/Users/bastien/Sites/kitten/spec/dummy/client/node_modules/react-cropper"